postgres/user_service.go

```diff
@@ -28,7 +28,7 @@ func (u *UserService) CreateUser(
 	email core.EmailAddress,
 	lang string,
 ) (*core.User, error) {
-	if email == nil {
+	if len(email.String()) == 0 {
```

Comment:
Deze check is eigenlijk niet meer nodig nu, een email met lengte 0 zou sowieso al ongeldig moeten zijn

Reply:
Ja, het zit er in owv de tests. Zoals ik al aangaf zijn er gevallen die zich niet mogen voordoen, maar die we toch in de test zetten. De test voor het aanmaken van een user moet falen met een ongeldige email

Reply:
Waarom zou je ooit extra validatie doen in een functie om een test te doen slagen die iets test dat eigenlijk nooit kan voorkomen? 😅